zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -348,39 +289,33 @@ fn rw_to_be_limbs(row: &Rw) -> Vec<u16> {
         .collect()
 }
 
-fn upper_limb_difference_possible_values<F: Field>(
-    cur: Queries<F>,
-    prev: Queries<F>,
-) -> Vec<Expression<F>> {
-    let mut result = vec![];
-    let mut partial_sum = 0u64.expr();
-    for (cur_limb, prev_limb) in cur.be_limbs()[..15].iter().zip(&prev.be_limbs()[..15]) {
-        partial_sum = partial_sum * (1u64 << 16).expr() + cur_limb.clone() - prev_limb.clone();
-        result.push(partial_sum.clone())
-    }
-    result
-}
-
-fn lower_limb_difference_possible_values<F: Field>(
+fn rlc_limb_differences<F: Field>(
```

Comment:
Note: at index `i`, we have the rlc of limb_differences from `0` to `i-1`.
At index `0` we have `0`.